core/src/main/java/nl/nn/adapterframework/pipes/ReplacerPipe.java

```diff
@@ -60,8 +60,8 @@ public void configure() throws ConfigurationException {
 			}
 			log.info("finds [{}] replaces with [{}]", getFind(), getReplace());
 			if (!StringUtils.isEmpty(getLineSeparatorSymbol())) {
-				find=replace(find,lineSeparatorSymbol,System.getProperty("line.separator"));
-				replace=replace(replace,lineSeparatorSymbol,System.getProperty("line.separator"));
+				find = find != null ? find.replace(lineSeparatorSymbol, System.getProperty("line.separator")) : null;
+				replace = replace != null ? replace.replace(lineSeparatorSymbol, System.getProperty("line.separator")) : null;
```

Comment:
Ik weet niet hoe volledig de ReplacePipeTests zijn, hoe was/is de dekking?

Reply:
90%, niet gedekt zijn wat uitzonderingen en een van de "Replace Non XML characters" branches. (Dekking gemeten over core unit tests + IAF test samen).

De code die de string-replacements deed was door IntelliJ identiek bevonden aan de code in StringUtil.replace, waarvan ik al eerder had gezien dat we die functioneel gelijk is aan String.replace() en de test-dekking daarvan is goed.